### ISSUE TITLE
Internal NBA js has changed and endpoints may have changed

### ISSUE THREAD

**Issue by seemethere** [seemethere/nba_py#67]
# Overview
The NBA continually changes there underlying API and so in order to keep up with it we need to keep sourcing their internal javascript files.

The latest one is located @ http://stats.nba.com/scripts/theme.min.js

**Comment by mwilliammyers** [comment]
Yeah, this is going to be a continual problem. Something like the idea proposed in #40 could help alleviate this...

**Comment by seemethere** [comment]
Yeah @mwilliammyers I was considering something like that for an `nba_py2` because I don't want to break others functionality on this module, but we could easily just put that in here as well.
